helpers/csrf.js

```diff
@@ -12,7 +12,19 @@ const duplicateTokenHandler = (req, res, next) => {
 	next();
 };
 
+const errorHandler = (err, req, res, next) => {
+	if (err.code !== 'EBADCSRFTOKEN') return next(err);
```

Comment:
change this: if EBADCSRFTOKEN then render, else return next... (this is bad order :) )